packages/websocket/install

```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
-
 set -x
 set -e
 
-# install libssl-dev for bionic image. No need for xenial.
```

Comment:
No longer true that it isn't needed for xenial?

Reply:
Good point, I'll test without and make sure.

Reply:
Turns out neither focal nor xenial need this.  I'll remove this entire package